minimesos/src/main/java/com/containersol/minimesos/main/CommandUp.java

```diff
@@ -209,6 +209,7 @@ private void updateWithParameters(ClusterConfig clusterConfig) {
         // Mesos Master
         MesosMasterConfig masterConfig = (clusterConfig.getMaster() != null) ? clusterConfig.getMaster() : new MesosMasterConfig();
         masterConfig.setImageTag(getMesosImageTag());
+        masterConfig.setLoggingLevel(clusterConfig.getLoggingLevel());
```

Comment:
I believe this is wrong: with changes in `MesosContainerConfig` users can specify logging level on container level. However this code will overwrite it with default or specified level from cluster. This one should work like "number of agents" - if was not specified on container level then take from cluster.